example/calculate_pi/calculate_pi.cpp

```diff
@@ -25,6 +25,9 @@
 
 using namespace parthenon::package::prelude;
 
+using parthenon::IndexDomain;
+using parthenon::IndexRange;
```

Comment:
Should these also get added to the `prelude` headers so they can be included in a single include call?

Reply:
Good idea.